cli/utils.go

```diff
@@ -1,17 +1,33 @@
 package cli
 
 import (
+	"bytes"
 	"encoding/json"
 	"fmt"
 	"os"
 )
 
-func PrettyPrintJSON(data interface{}) {
-	s, err := json.MarshalIndent(data, "", "  ")
+func PrettyJSON(data interface{}) string {
+	var s []byte
+	var err error
+
+	switch data.(type) {
+	case []byte:
+		var pretty bytes.Buffer
+		err = json.Indent(&pretty, data.([]byte), "", "  ")
+		s = pretty.Bytes()
+	case string:
+		var pretty bytes.Buffer
+		err = json.Indent(&pretty, []byte(data.(string)), "", "  ")
+		s = pretty.Bytes()
+	default:
+		s, err = json.MarshalIndent(data, "", "  ")
+	}
```

Comment:
that seems too complicated than it needs to be, can you explain in person why, please?